Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -71,36 +67,22 @@ namespace PhysX
         }
     }
 
-    void EditorProxyAssetShapeConfig::Reflect(AZ::ReflectContext* context)
+    // O3DE_DEPRECATION_NOTICE(GHI-14718)
```

Comment:
nit: You can leave this but it's generally fine to leave API deprecation comments in the .h files that are public (rather than repeating in multiple places in the implementation)